### PR TITLE
Features/wadnr 1914 external map layers

### PR DESCRIPTION
**Pull request by MCSpelman**

### CODE REVIEW

File: Source/ProjectFirma.Web/Views/MapLayer/EditExternalMapLayerViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        [StringLength(ExternalMapLayer.FieldLengths.DisplayName)]
+        [DisplayName("Display Name")]
+        public string DisplayName { get; set; }
+
+        [Required]
+        [StringLength(ExternalMapLayer.FieldLengths.LayerUrl)]
+        [DisplayName("Url")]
+        public string LayerUrl { get; set; }
+
+        [Required]
```
Comment (tomkamin) [inline Source/ProjectFirma.Web/Views/MapLayer/EditExternalMapLayerViewModel.cs:47]: Lets drop the Required on the booleans. Feels weird to me to see a required mark on a checkbox in a form.

**Review by tomkamin — CHANGES_REQUESTED** [review]
Just a small update to the form to remove the required from bools. 
Everything looks great. I tested out the tile layer and didn't encounter any issues with the map on my machine. One of the not tiled layers I tested took a while to load, but it was huge! Lots of shapes all over the west coast. Slowed my browser way down.

File: Source/ProjectFirma.Web/Views/MapLayer/EditExternalMapLayerViewModel.cs (hunk 1 above)
Comment (MCSpelman) [reply Source/ProjectFirma.Web/Views/MapLayer/EditExternalMapLayerViewModel.cs:47]: fixed

**Review by tomkamin — APPROVED** [review]
Thanks!

**Pull request merged**